web_programming/get_top_billioners.py

```diff
@@ -0,0 +1,84 @@
+"""
+CAUTION: You may get a json.decoding error.  This works for some of us but fails for others.
+"""
+
+from datetime import datetime
+
+import requests
+from rich import box
+from rich import console as rich_console
+from rich import table as rich_table
+
+LIMIT = 10
+TODAY = datetime.now()
+
+API_URL = (
+    "https://www.forbes.com/forbesapi/person/rtb/0/position/true.json"
+    "?fields=personName,gender,source,countryOfCitizenship,birthDate,finalWorth"
+    f"&limit={LIMIT}"
+)
```

Comment:
`requests` can automatically encode the URL parameters: https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls. Please use this instead.